boards/xtensa/intel_s1000_crb/doc/intel_s1000.rst

```diff
@@ -0,0 +1,216 @@
+tap the UART data. The J8 Header on S1000 CRB is dedicated for UART.
+Please do the connection between the J8 header and UART chip as shown below.
+
++------------+-----------+
+| UART chip  | J8 Header |
++============+===========+
+| DTR        |           |
++------------+-----------+
+| RX         | 2         |
++------------+-----------+
+| TX         | 4         |
++------------+-----------+
+| VCC        |           |
++------------+-----------+
+| CTS        |           |
++------------+-----------+
+| GND        | 10        |
++------------+-----------+
+
+We use minicom for monitoring the UART data. Follow the below steps
+to configure minicom.
+
+Attach one end of the USB cable to the UART chip and the other end to the
+Linux system. Type dmesg | grep USB in the terminal. This will show the
+tty to which the USB is connected. If the tty connected is ttyUSB0, type
+minicom -D /dev/ttyUSB0 in the terminal. This will open the minicom. The
+serial settings configured in zephyr is "115200 8N1". This is also the
+default settings in minicom and can be verified by pressing Ctrl-A Z P.
+
+Using JTAG
+==========
+
+We use a flyswatter2 debugger to connect to S1000 CRB. The pinouts for
+flyswatter2 and the corresponding pinouts for CRB are shown below.
+Note that pin 6 on CRB is left unconnected.
+
+The corresponding pin mapping is
++-------------+-----------+
+| Flyswatter2 |   S1000   |
++=============+===========+
+|     1       |     7     |
++-------------+-----------+
+|     2       |    NC     |
++-------------+-----------+
+|     3       |     4     |
++-------------+-----------+
+|     4       |    NC     |
++-------------+-----------+
+|     5       |     3     |
++-------------+-----------+
+|     6       |     8     |
++-------------+-----------+
+|     7       |     2     |
++-------------+-----------+
+|     8       |    NC     |
++-------------+-----------+
+|     9       |     1     |
++-------------+-----------+
+|     10      |    NC     |
++-------------+-----------+
+|     11      |    NC     |
++-------------+-----------+
+|     12      |    NC     |
++-------------+-----------+
+|     13      |     5     |
++-------------+-----------+
+|     14      |    NC     |
++-------------+-----------+
+|     15      |    NC     |
++-------------+-----------+
+|     16      |    NC     |
++-------------+-----------+
+|     17      |    NC     |
++-------------+-----------+
+|     18      |    NC     |
++-------------+-----------+
+|     19      |    NC     |
++-------------+-----------+
+|     20      |    NC     |
++-------------+-----------+
+
+Ideally, these connections should have been enough to get the debug working.
+However, we need to short 2 pins on Host Connector J3 via a 3.3k resistor
+(simple shorting without the resistor will also do) for debugging to work.
+Those 2 pins are Pin5 HOST_RST_N_LT_R) and Pin21 (+V_HOST_3P3_1P8).
+
+References
+**********
+
```

Comment:
If you intended for these links to show up in the generated output, add this directive before them:

    .. target-notes::

and a blank line.